api/src/main/java/org/apache/iceberg/OverwriteFiles.java

```diff
@@ -154,4 +154,14 @@ public interface OverwriteFiles extends SnapshotUpdate<OverwriteFiles> {
    * @return this for method chaining
    */
   OverwriteFiles validateNoConflictingDeletes();
+  /**
+   * Append a {@link ManifestFile} to the table. Similar to {@link
+   * AppendFiles#appendManifest(ManifestFile)} Having this with {@link OverwriteFiles} too, helps to
+   * Use manifestFile with data files. It will put less pressure on memory in case of thousands of
+   * data files together.
+   *
+   * @param file a manifest file
+   * @return this for method chaining
+   */
```

Comment:
I think cases and grammar has to be fixed in the above java doc.

How about copy pasting the java doc from `AppendFiles#appendManifest()`?

Reply:
I am also thinking we should call it as `appendDataManifest`, since it doesn't support delete manifests.